src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -316,7 +318,12 @@ void AdvertiserMinMdns::Clear()
 
 CHIP_ERROR AdvertiserMinMdns::Advertise(const OperationalAdvertisingParameters & params)
 {
-    Clear();
+    /* NOTE :
+     * Clear() method should not be called in Advertise()
+     * because for the devices which call Advertise() method more than 1 times i.e. advertise operational, commissionable, and
+     * commissioner needs the result to be additive. If Clear() is called at the beginning of these Advertise methods, then only the
+     * last call to Advertise will take effect.
```

Comment:
```suggestion
     * Clear() should not be called in Advertise() because some devices want to advertise
     * multiple things (operational advertisement, commissionable advertisement, commissioner
     * advertisement).   If Clear() were called at the beginning of these Advertise methods, then
     * only the last call to Advertise would take effect.
```